### DISCUSSION TITLE
clients can connect to relay server with Pubilc IP,but can't connect to relay server with domain name.

### DISCUSSION THREAD

**Discussion by toyqiu** [rustdesk/rustdesk-server#356] [General]
If you can directly connect to the relay server using the public IP address, it indicates that there are no issues with the router, port, or firewall settings. I have tested and confirmed that my domain name is indeed pointing to the current public IP address. However, when I enter the public IP address + port number on the client side, it connects successfully, but when I enter the domain name + port number, it fails to connect. What could be the problem?

**Comment by pdifeo** [comment]
https://github.com/rustdesk/rustdesk-server-pro/issues/102#issuecomment-1745344975

**Comment by dinger1986** [comment]
Try domain name only if using a standard port
